playbooks/inventory/dns-server/group_vars/all.yml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Why are we putting an inventory in the playbooks directory?

Reply:
Yeah, good point. It's a bit of a struggle to "pair-up" the proper inventory with the correct playbook. But it should go to the top level. Maybe we just need a note in some READMEs for which inventory goes with what playbook(s).